src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +61,24 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Override equality, so conversations are compared based on their ID */
+  @Override
+  public boolean equals(Object o) {
+
+    if (o == this) return true;
```

Comment:
This logic is redundant with that below, we can probably omit it

Reply:
Done.